nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -298,87 +377,223 @@ def permuted_ols(
-        `model_intercept`)
+        ``model_intercept``).
 
-    model_intercept : bool, optional
+    model_intercept : :obj:`bool`, optional
         If True, a constant column is added to the confounding variates
         unless the tested variate is already the intercept.
-        Default=True
+        Default=True.
 
-    n_perm : int, optional
+    n_perm : :obj:`int`, optional
         Number of permutations to perform.
         Permutations are costly but the more are performed, the more precision
-        one gets in the p-values estimation. Default=10000.
+        one gets in the p-values estimation.
+        If ``n_perm`` is set to 0, then no p-values will be estimated.
+        Default=10000.
 
-    two_sided_test : boolean, optional
+    two_sided_test : :obj:`bool`, optional
         If True, performs an unsigned t-test. Both positive and negative
         effects are considered; the null hypothesis is that the effect is zero.
         If False, only positive effects are considered as relevant. The null
         hypothesis is that the effect is zero or negative. Default=True.
 
-    random_state : int or None, optional
+    random_state : :obj:`int` or None, optional
         Seed for random number generator, to have the same permutations
         in each computing units.
 
-    n_jobs : int, optional
+    n_jobs : :obj:`int`, optional
         Number of parallel workers.
         If -1 is provided, all CPUs are used.
         A negative number indicates that all the CPUs except (abs(n_jobs) - 1)
         ones will be used. Default=1.
 
-    verbose : int, optional
+    verbose : :obj:`int`, optional
         verbosity level (0 means no message). Default=0.
 
+    masker : None or :class:`~nilearn.maskers.NiftiMasker` or \
+            :class:`~nilearn.maskers.MultiNiftiMasker`, optional
+        A mask to be used on the data.
+        This is required for cluster-level inference, so it must be provided
+        if ``threshold`` is not None.
+
+        .. versionadded:: 0.9.2
+
+    threshold : None or :obj:`float`, optional
+        Cluster-forming threshold in p-scale.
+        This is only used for cluster-level inference.
+        If None, cluster-level inference will not be performed.
+        Default=None.
+
+        .. warning::
+
+            Performing cluster-level inference will increase the computation
+            time of the permutation procedure.
+
+        .. versionadded:: 0.9.2
+
+    output_type : {'legacy', 'dict'}, optional
+        Determines how outputs should be returned.
+        The two options are:
+
+        -   'legacy': return a pvals, score_orig_data, and h0_fmax.
+            This option is the default, but it is deprecated until 0.13,
+            when the default will be changed to 'dict'.
+            It will be removed in 0.15.
+        -   'dict': return a dictionary containing output arrays.
+            This option will be made the default in 0.13.
+
+        .. deprecated:: 0.15
```

Comment:
```suggestion
        .. deprecated:: 0.9.2
```

(otherwise it renders as "deprecated since 0.15".)

Reply:
🤦 thanks for catching this! I've fixed this and the other instances in 2ab34d0.